pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>io.percy</groupId>
   <artifactId>percy-appium-app</artifactId>
-  <version>2.0.0-beta.0-SNAPSHOT</version>
+  <version>2.0.1-beta.1-SNAPSHOT</version>
```

Comment:
```suggestion
  <version>2.0.0-beta.1-SNAPSHOT</version>
```

Reply:
@itsjwala  its released properly only. The JAR version mentioned here generally shows the next version applicable.
https://central.sonatype.com/artifact/io.percy/percy-appium-app/2.0.0-beta.1

Reply:
Ideally we don't need this then, we don't need to store future version, we can update this anytime when we release new version